### DISCUSSION TITLE
对于”Gain x 状态“：[对自身施加] 或 [使自身获得] 的翻译明晰

### DISCUSSION THREAD

**Discussion by N071** [LocalizeLimbusCompany/LocalizeLimbusCompany#49] [常规 | General]
如题，对于”Gain“的翻译目前有两种不同的译法：[对自身施加] 或 [使自身获得]（即便英文原文的表达方式一样）。
将在后续把图片发上来。

---
整理目前已知的情报：
CN现行的翻译：
- 有关强度动词用”使目标(自身)增加“
- 有关层数动词用”对目标(自身)施加“ 以及偶有特殊用法“使自身获得”

JP：
- 有关强度动词用”付予“
- 有关层数(JP:回数)动词用”增加“
- JP没有类似“对”和“使”的区分。

EN：
- 不论强度或层数，统一使用动词“Inflict”。
- 特殊情况：凡是对自身造成效果的，不论强度或层数，统一使用动词“Gain”

可见，CN的大体思路与JP相同，均是**通过“强度”和“层数”来**区分动词
对于EN：
EN没有因“强度”或“层数”区别而进行动词的区分。
与其它语言的思路不同，EN**通过目标的不同**区分了动词。

因此 综合来看，对于“使自身获得”这类特殊用法，是否应该放弃EN的思路，此处与JP统一，用“对自身施加” 是否更加妥当？
另：因为CN语言的特殊性，即便JP没有类似“对”和“使”的区分，CN的“对”和“使”的区分我仍认为应当保留(即服从现行CN翻译)。

**Comment by N071** [comment]
注：类似的例子还有很多，此处以CN_Skills_Ego.json为例
![image](https://user-images.githubusercontent.com/128023427/231341047-ecc9eb7b-d17a-4727-a5f9-434e630dd23e.png)

---
![image](https://user-images.githubusercontent.com/128023427/231341095-2915e202-d6c0-441f-bf0a-e443ec52c540.png)

**Comment by N071** [comment]
另外的提议是延续英文的使用方法，将 [对自身施加] 与 [使自身获得] 全部统一为更加简明的 [获得]

**Comment by Karyut** [comment]
个人看法：“使”作用于自身及友军，“对”作用于敌人
“获得”作用于buff，“施加”作用于debuff
